### ISSUE TITLE
Inventorty doesn't work in creative mode on Forge

### ISSUE THREAD

**Issue by beholderface** [SamsTheNerd/HexGloop#25]
Forge version 43.3.5

https://github.com/SamsTheNerd/HexGloop/assets/13290805/d9370f65-2830-47fe-a315-2dabc6ad2ae4

**Comment by SamsTheNerd** [comment]
yeah, it seems like the onClick and onClicked method just isn't called on the server in client. I have no idea why but probably something with trusting the creative client more with handling its inventory ? 

I added a note about it in the docs for now atleast.
